.studiorc

```diff
@@ -208,11 +208,16 @@ _stop-builder() {
 stop-builder() { stop-on-failure _stop-builder "$@"; }
 
 generate_bldr_keys() {
-  KEY_NAME=$(hab user key generate bldr | grep -Po "bldr-\d+")
-  for svc in api jobsrv worker; do
-    hab file upload "builder-${svc}.default" $(date +%s) "/hab/cache/keys/${KEY_NAME}.pub"
-    hab file upload "builder-${svc}.default" $(date +%s) "/hab/cache/keys/${KEY_NAME}.box.key"
-  done
+  if ls /hab/cache/keys/bldr-*.pub 1> /dev/null 2>&1; then
+    echo "Re-using existing builder key"
+  else
+    echo "Generating builder key"
+    KEY_NAME=$(hab user key generate bldr | grep -Po "bldr-\d+")
+    for svc in api worker; do
+      hab file upload "builder-${svc}.default" $(date +%s) "/hab/cache/keys/${KEY_NAME}.pub"
+      hab file upload "builder-${svc}.default" $(date +%s) "/hab/cache/keys/${KEY_NAME}.box.key"
```

Comment:
I know these were like this and you're just changing the indentation, but would you mind double-quoting the `$(date +%s)` calls? I'm trying to work towards resolving all our lints for https://github.com/habitat-sh/habitat/issues/4170

Reply:
Sure thing - will update.